data/oot/world/overworld.yml

```diff
@@ -289,21 +290,27 @@
     "Goron City Medigoron Giant Knife": "is_adult && has(WALLET)"
     "Goron City GS Platform": "is_adult && has_ranged_weapon"
     "Goron City GS Maze": "is_child && has_explosives"
-"Zora River":
+"Zora River Front":
   exits:
     "Hyrule Field": "true"
+    "Zora River": "is_adult || (is_child && has_explosives)"
+  locations:
+    "Zora River GS Tree": "is_child"
+"Zora River":
+  exits:
+    "Zora River Front": "true"
     "Zora Domain": "can_play(SONG_ZELDA)"
+    "Lost Woods": "can_dive_small"
```

Comment:
The way back should also be added in Lost Woods

Reply:
It is I think, line 155